client/lxd_instances.go

```diff
@@ -2087,6 +2087,80 @@ func (r *ProtocolLXD) DeleteInstanceLogfile(name string, filename string) error
 	return nil
 }
 
+// GetInstanceExecOutputLogFiles returns a list of logfiles for the commands ran inside an instance.
+func (r *ProtocolLXD) GetInstanceExecOutputLogFiles(name string) ([]string, error) {
+	path, _, err := r.instanceTypeToPath(api.InstanceTypeAny)
```

Comment:
Here we need to check if the `container_exec_recording` extension is available
```golang
if !r.HasExtension("container_exec_recording") {
    return nil, fmt.Errorf("The server is missing the required \"container_exec_recording\" API extension")
}
```

Reply:
Please use r.CheckExtension()